dangerfile.js

```diff
@@ -18,9 +18,46 @@ const {danger, warn, fail} = require('danger')
 
 // Danger.js documentation: http://danger.systems/js/
 
+// Check for PRs having a [ ] at the beginning of their title.
+
 if (!danger.github.pr.title.startsWith("[")) {
   warn('This PR title does not include an affected component.'
     + ' For example: "[SomeComponent] Title. If something is affecting multiple components and the'
     + ' change can\'t be broken up into multiple PRs, then the common trait of the change could be'
     + ' used in the PR title. E.g. `[documentation]`.');
 }
+
+// Check for PRs that modify multiple components.
+
+let all_files = danger.git.modified_files
+  .concat(danger.git.deleted_files)
+  .concat(danger.git.created_files);
+
+// Extract any component source modifications.
+let component_files = all_files.filter(function(path) {
+  return path.startsWith('components/') && path.indexOf('/src') >= 0;
+});
+
+// Generate a list of unique components that have been modified.
+let components = Array.from(new Set(component_files.map(function(path) {
+  var path_parts = path.split('/');
+  path_parts.splice(0, 1); // Drop the components/ prefix.
+  
+  var component_path = [];
+  // Convention: lower-case names are directories. Upper-case names are components.
```

Comment:
Where is this convention used? In the source, we use lowercase and Camel Case, so it's `components/ActivityIndicator`, not `components/ACTIVITYINDICATOR`.

Reply:
Updated the docs to more clearly document the convention.